api/types.proto

```diff
@@ -299,20 +324,27 @@ message UpdateStatus {
 		UPDATING = 1;
 		PAUSED = 2;
 		COMPLETED = 3;
-		// TODO(aaronl): add ROLLING_BACK, ROLLED_BACK as part of
-		// rollback support.
+		ROLLBACK_STARTED = 4;
+		ROLLBACK_PAUSED = 5; // if a rollback fails
+		ROLLBACK_COMPLETED = 6;
 	}
 
 	// State is the state of this update. It indicates whether the
-	// update is in progress, completed, or is paused.
+	// update is in progress, completed, paused, rolling back, or
+	// finished rolling back.
 	UpdateState state = 1;
 
 	// StartedAt is the time at which the update was started.
 	Timestamp started_at = 2;
 
-	// CompletedAt is the time at which the update completed.
+	// CompletedAt is the time at which the update completed successfully,
+	// paused, or finished rolling back.
 	Timestamp completed_at = 3;
 
+	// RollbackStartedAt is the time at which the update started rolling
+	// back to the previous version.
+	Timestamp rollback_started_at = 5;
```

Comment:
Also, we had this conversation a while ago (at the same of adding pause), don't remember the outcome but: does it make sense to add timestamp fields every time we add a new state?

Wouldn't it help to have a list of `UpdateStatus`es and each time we transition to a new state, we add another item in order to keep history?

The same would apply to task status, node status, and so on.

For instance, if we had this, UX wise instead of displaying events like this:
```
Started At: 2016-09-01T00:19:32
Rollback At: 2016-09-01T00:19:34
Completed At: 2016-09-01T00:19:36
```

We could do something like this:
```
Events:
TIME                       EVENT               ERROR
2016-09-01T00:19:32        Update Started      
2016-09-01T00:19:34        Rollback Started    Unable to find image "foo"
2016-09-01T00:19:36        Rollback Completed      

```